Dockerfile

```diff
@@ -23,6 +23,9 @@ USER $USER
 ENV ANDROID_SDK_ROOT /home/${USER}/android-sdk
 RUN mkdir ${ANDROID_SDK_ROOT}
 WORKDIR $ANDROID_SDK_ROOT
+
+# If you change the version here (6200805, meaning r22b/22.1.7171670),
```

Comment:
Are you sure 6200805 means r22b/22.1.7171670? 6200805 seems to be a version of [Command line tools](https://developer.android.com/studio/command-line)
Currently on page https://developer.android.com/studio the latest version of command line tools is commandlinetools-linux-8512546_latest.zip

How does this number maps to SDK and NDK version?

Reply:
Currently command line tools version is commandlinetools-linux-8512546_latest.zip and latest NDK version is r23c aka 23.2.8568313. So it seems there is a correspondnce to 8568313, but I can't find NDK version with 6200805 in its version on the page with old versions: https://github.com/android/ndk/wiki/Unsupported-Downloads

Reply:
In the output of `sdkmanager --sdk_root=/home/deltachat/android-sdk/ --list` inside docker container:
```
  ndk-bundle                                                                               | 22.1.7171670 | NDK                                                                 
  ndk;16.1.4479499                                                                         | 16.1.4479499 | NDK (Side by side) 16.1.4479499                                     
  ndk;17.2.4988734                                                                         | 17.2.4988734 | NDK (Side by side) 17.2.4988734                                     
  ndk;18.1.5063045                                                                         | 18.1.5063045 | NDK (Side by side) 18.1.5063045                                     
  ndk;19.2.5345600                                                                         | 19.2.5345600 | NDK (Side by side) 19.2.5345600                                     
  ndk;20.0.5594570                                                                         | 20.0.5594570 | NDK (Side by side) 20.0.5594570                                     
  ndk;20.1.5948944                                                                         | 20.1.5948944 | NDK (Side by side) 20.1.5948944                                     
  ndk;21.0.6113669                                                                         | 21.0.6113669 | NDK (Side by side) 21.0.6113669                                     
  ndk;21.1.6352462                                                                         | 21.1.6352462 | NDK (Side by side) 21.1.6352462                                     
  ndk;21.2.6472646                                                                         | 21.2.6472646 | NDK (Side by side) 21.2.6472646                                     
  ndk;21.3.6528147                                                                         | 21.3.6528147 | NDK (Side by side) 21.3.6528147                                     
  ndk;21.4.7075529                                                                         | 21.4.7075529 | NDK (Side by side) 21.4.7075529                                     
  ndk;22.0.7026061                                                                         | 22.0.7026061 | NDK (Side by side) 22.0.7026061                                     
  ndk;22.1.7171670                                                                         | 22.1.7171670 | NDK (Side by side) 22.1.7171670                                     
  ndk;23.0.7599858                                                                         | 23.0.7599858 | NDK (Side by side) 23.0.7599858                                     
  ndk;23.1.7779620                                                                         | 23.1.7779620 | NDK (Side by side) 23.1.7779620                                     
  ndk;23.2.8568313                                                                         | 23.2.8568313 | NDK (Side by side) 23.2.8568313                                     
  ndk;24.0.8215888                                                                         | 24.0.8215888 | NDK (Side by side) 24.0.8215888
```

Seems command line tools version at this line has nothing to do with NDK version. It's just that `ndk-bundle` currently points to 22.1.7171670. I guess it makes sense to pin `'ndk;22.1.7171670'` instead of `ndk-bundle` in the Dockerfile. Or maybe something newer.

Reply:
Also here for F-Droid NDK r22b is used currently: https://gitlab.com/fdroid/fdroiddata/-/blob/master/metadata/com.b44t.messenger.yml#L1398